libycp/src/Import.cc

```diff
@@ -108,6 +108,15 @@ Import::import (const string &name, Y2Namespace *preloaded_namespace)
 	    }
 	}
 
+        // for runtime error during loading lets act like if module is there, just error namespace is created
+        if (isErrorNamespace(name_space))
+        {
+            ycp2error("Runtime error when loading module '%s'", cname);
+            m_active_modules.insert (std::make_pair (m_name, name_space));			// insert to list of known modules
+	    m_module = m_active_modules.find (m_name);
+            return 0;
```

Comment:
and that is problem as in this case it won't store namespace, which we want and it also do not cache this runtime dependency. And last but not least if your return "-1" then it tries to load it somewhere else, which is exactly what I do not want, as module exists and just have runtime problem to load.